tests/AbstractTestCase.php

```diff
@@ -32,4 +32,11 @@ protected function assertDqlWhere(?array $expectedDqlWhere, QueryBuilder $queryB
             $this->assertSame($expectedDqlWhere, $where->getParts());
         }
     }
+
+    protected function createBlankCallback(string $name): callable
```

Comment:
You expect this to be useful in other test cases. If not i would place it in `FilterFunctionTest` directly.

Reply:
yes it will be, but by these partial pull requests, it is not that obvious now..